### ISSUE TITLE
send delay message, but deliver immediately

### ISSUE THREAD

**Issue by ilaipi** [apache/pulsar-client-node#135]
pulsar version: 2.7.0, run in docker by docker-compose
client version: 1.2.0

OS: wsl2 ubuntu

```
// producer:
send() {
    const producer = await this.pulsar.createProducer({
      topic: 'my-topic',
    });
    await producer.send({
      data: Buffer.from(`Hello, pulsar, ${Date.now()}`),
    });

    await producer.send({
      data: Buffer.from(`Hello, pulsar, delayed, ${Date.now()}`),
      deliverAfter: 30000,
      // deliverAt: new Date().getTime() + 50000,
    });
    await producer.close();
}

// consumer
    await this.pulsar.subscribe({
      topic: 'my-topic',
      subscription: 'my-subscription',
      subscriptionType: 'Shared',
      listener: (msg, msgConsumer) => {
        console.log(msg.getData().toString());
        msgConsumer.acknowledge(msg);
      },
    });
```

when I call send(), the consumer console print 
```
Hello, pulsar, 1607572676764
Hello, pulsar, delayed, 1607572676776
 ```
immediately.

Is this right?

**Comment by hrsakai** [comment]
v1.2.0 doesn't include this feature yet.
We will release v1.3.0 in a few months. Please wait for it.

**Comment by ericshao** [comment]
@hrsakai does this fixed in v1.3.0?

**Comment by hrsakai** [comment]
@ericshao 
Yes, v1.3.0 supports for delayed delivery.

**Comment by hrsakai** [comment]
@ilaipi 
 We released v1.3.0, so closed this issue.

**Comment by immatt2015** [comment]
pulsar-client ^1.9.0
